app.go

```diff
@@ -117,7 +121,7 @@ func (app *App) RunEventProcessor(ctx context.Context, wg *sync.WaitGroup) {
 			for _, event := range eventMessage.Events {
 				go app.processEvent(event)
 			}
-			offset := eventMessage.Events[len(eventMessage.Events)-1].Offset + 1
+			offset := eventMessage.Offset + 1
```

Comment:
why ?

Reply:
eventMessage.Offset is an offset of the last message in the pack